src/anupSJun20/FindItemIDfromAString.java

```diff
@@ -0,0 +1,42 @@
+package anupSJun20;
+/*Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/temId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+
+public class FindItemIDfromAString {
+	void findItemID(String str) {
+		System.out.println("\nGiven String is : "+str);
+		String temp = "";
+		try {
+			String newstr = str.substring(str.indexOf("item"));
+			if (newstr.contains("item")) {
+				for (int index = 0; index < newstr.length(); index++) {
```

Comment:
If condition is correct
One change is, in for loop we can starts from newstr.Indexof("item" )+4 this will return ASCII value
4 for item length, 4 for we want to skip to check "item" word

Rest logic is correct